dev/aarch64_opt/src/README.md

```diff
@@ -0,0 +1,62 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# mlkem-native AArch64 back-end SLOTHY-optimized code
+
+This directory contains the AArch64 back-end _after_ it has
+been optimized by SLOTHY.
+
+## Running SLOTHY from the Makefile
+
+If the "clean" back-end sources in `../../aarch64_clean/src/*.S` change,
+then the `Makefile` in this directory can be used to re-generate the
+optimized sources using SLOTHY.
+
+The Makefile requires a _particular_ set up. In short:
+
+1. Do NOT run from within the NIX environment.
+2. You must have Python 3.11 on PATH (3.12 does not work at present)
+3. You must have the LLVM tool `llvm-mc` on PATH.
+4. You must have the slothy-cli on PATH.
+5. You must start the SLOTHY Python virtual-environment.
```

Comment:
```suggestion
The Makefile requires you to have a working SLOTHY setup. Note that mlkem-native's nix shell does currently not include SLOTHY.
```

Reply:
If the README in https://github.com/slothy-optimizer/slothy/ is out of date, let's update/improve that.

Reply:
Noted. I will check it and raise an Issue there if necessary.